.github/actions/publish-release-contents/action.yml

```diff
@@ -32,4 +32,4 @@ runs:
       uses: actions/download-artifact@v2
       with:
         name: bff
-        path: ${{ inputs.RELEASE_FOLDER_PATH }}/functions/bff
\ No newline at end of file
+        path: ${{ inputs.RELEASE_FOLDER_PATH }}/functions/bff
```

Comment:
Question: Is this change intended?

Reply:
I don't know why it appears. It wasn't intended at least. Must be a change only git cam see.